client/extensions/woocommerce/state/sites/settings/email/queryLists.jsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
🤔  I know many of the components in the `woocommerce` extension use `.js` as their file extension. Not sure if we should stick to that here or ... ?

Reply:
From slack, I can see that we do not care in the end :)